basc_py4chan/thread.py

```diff
@@ -60,6 +61,10 @@ def imagelimit(self):
     def bumplimit(self):
         return self.topic._data.get('bumplimit') == 1
 
+    @property
+    def custom_spoiler(self):
+        return self.topic._data.get('custom_spoiler')
```

Comment:
Hmm, since we're already using `.get` here, perhaps we should `.get('custom_spoiler', 0)`?